### PR TITLE
Task/change state identifiers

### LINKED ISSUE #50: Create the Status literals
Current status literals are spreaded through the code, and do not correspond with the final literals agreed with the client:

P -> Pending
C.S -> Success
C.E -> Fail
X -> Closed

### PR DESCRIPTION
**Pull request by dmoranj**
Fixes #50

### CODE REVIEW

**Comment by AlvaroVega** [comment]
LGTM

File: lib/constants.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+ * If not, seehttp://www.gnu.org/licenses/.
+ *
+ * For those usages not covered by the GNU Affero General Public License
+ * please contact with::[iot_support@tid.es]
+ */
+var constants = {
+    states: {
+        PENDING: 'P',
+        SUCCESS: 'C.S',
+        FAILED: 'C.E',
```
Comment (gtorodelvalle) [inline lib/constants.js]: Wasn't this one `C.F` (from failure, I guess) instead of `C.E`? :)

Comment (dmoranj) [inline lib/constants.js]: Fixed in f249711

**Comment by gtorodelvalle** [comment]
LGTM and merging :)

**Pull request merged**